doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -50,12 +50,12 @@ following Features tabs defined here.
 | Conformance                  | X             | Kubernetes conformance test |
 | None                         | X             | Kubernetes mainstream features |
 | Feature:DaemonSetUpdateSurge |               | Daemon set should surge pods onto nodes when spec was updated and update strategy is RollingUpdate |
-| Feature:IndexedJob           |               | Should create pods for an Indexed job with completion indexes |
+| Feature:[IndexedJob](https://kubernetes.io/docs/concepts/workloads/controllers/job/)          |               | Should create pods for an Indexed job with completion indexes |
 | Feature:StatefulSet          |               | Should creating a working zookeeper cluster |
 | Feature:StatefulUpgrade      |               | Stateful upgrade should maintain a functioning cluster |
```

Comment:
@collivier @CsatariGergely "https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies"?

Reply:
I thik this case is more about an concrete upgrare of a stateful application.